mtz-wizard.html

```diff
@@ -149,10 +149,10 @@
        * @param {HTMLElement[]} [steps = this.steps]
        * @return {Boolean}
        */
-      validate(steps = this.steps) {
-        return (steps || []).reduce((prev, step) => {
-          return prev && (step.validate ? step.validate() : true);
-        }, true);
+      async validate(steps = this.steps) {
+        return (await Promise.all((steps || []).map((step) => {
+          return step.validate ? step.validate() : Promise.resolve(true);
+        }))).every((stepIsValid) => stepIsValid);
```

Comment:
When doing a pure boolean check you can just use `Boolean`: `.every(Boolean);` this is because the callback function takes exactly 1 param and needs to return a boolean value, the `Boolean` type casting has the same requirements.